src/index.html

```diff
@@ -2,11 +2,355 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin ="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="./images/logo.png" alt="MOYO logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link "
+              href="#"
+              data-qa="nav-hover"
+            >
+               Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__catalog">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__description">
+            Product code: 195434
+          </p>
+          <div class="card__product-rating">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__price-container">
+            <h3 class="card__price-text">Price:</h3>
+            <p class="card__price-value">$2,199</p>
+          </div>
+          <a
+            href="#"
+            class="card__button-buy"
+            type="button"
+            data-qa="card-hover"
+          >
+          Buy
+          </a>
```

Comment:
```suggestion
          <a
            href="#"
            class="card__button-buy"
            type="button"
            data-qa="card-hover"
          >
            Buy
          </a>
```
Don't forget about formatting